src/datatypeconvertor.js

```diff
@@ -202,6 +202,9 @@ $.DataTypeConvertor = class {
             const img = new Image();
             img.onerror = img.onabort = reject;
             img.onload = () => resolve(img);
+            if (tile.tiledImage && tile.tiledImage.crossOriginPolicy) {
```

Comment:
Attempt to respect this property, since it is now used to actually download the image data.